### PR TITLE
Bump docs version for 7.0.0

### PR DESCRIPTION
**Pull request by urso**

### CODE REVIEW

File: libbeat/docs/version.asciidoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
-:stack-version: 7.0.0-rc2
+:stack-version: 7.0.0
 :doc-branch: 7.0
 :go-version: 1.11.5
-:release-state: prerelease
+:release-state: release
```
Comment (dedemorton) [inline libbeat/docs/version.asciidoc]:
```suggestion
:release-state: released
```

**Comment by ruflin** [comment]
jenkins, test this

**Comment by ruflin** [comment]
This needs a `make update`.

**Comment by urso** [comment]
Problem is the docker versions used by the k8s configs are validated against the docs version. I did try to split it up into 2 PRs, so that we can merge the docs and k8s updates in isolation (we've been asked to freeze the docs changes early).

Running `make update` and closing the other PR :(

**Pull request merged**